examples/check_integration_error.py

```diff
@@ -62,20 +66,11 @@
     data.ft("t2", shift=True)
     # {{{
     data /= sqrt(ndshape(data)["t2"]) * dt
-    error_pathway = (
-        set((
-            (j, k)
-            for j in range(ndshape(data)["ph1"])
-            for k in range(ndshape(data)["ph2"])
-        ))
-        - set(excluded_pathways)
-        - set([(signal_pathway["ph1"], signal_pathway["ph2"])])
-    )
-    error_pathway = [{"ph1": j, "ph2": k} for j, k in error_pathway]
     s_int, frq_slice = integral_w_errors(
         data,
-        signal_pathway,
-        error_pathway,
+        sig_path=signal_pathway,
```

Comment:
already discussed how we don't want to define signal pathway, and only want to define the excluded pathways.  assuming it's likely other comments haven't been addressed, and stopping here

Reply:
to be consistent with other code in the repo
, this should be called `signal_pathway`

Reply:
checked